students/jeff_shabani/session03/list_lab.py

```diff
@@ -3,63 +3,69 @@
 
 FRUIT_LIST = ['Apples', 'Pears', 'Oranges', 'Peaches']
```

Comment:
also remember that this isn't a constant so it shouldn't be in all caps.  Its a constantly changing list.  Also constants should use an immutable data structure like a tuple rather than a list.